lib/msal-browser/docs/configuration.md

```diff
@@ -103,7 +102,7 @@ See [Caching in MSAL](./caching.md) for more.
 | `iframeHashTimeout`        | Timeout in milliseconds to wait for iframe operations to resolve.                                                                                                                                                                                                                                             | integer (milliseconds)               | `6000`                                           |
 | `loadFrameTimeout`         | Timeout in milliseconds to wait for iframe/popup operations resolve. If provided, will set default values for `windowHashTimeout` and `iframeHashTimeout`.                                                                                                                                                    | integer (milliseconds)               | `undefined`                                      |
 | `navigateFrameWait `       | Delay in milliseconds to wait for the iframe to load in the window.                                                                                                                                                                                                                                           | integer (milliseconds)               | In IE or Edge: `500`, in all other browsers: `0` |
-| `asyncPopups`              | Sets whether popups are opened asynchronously. When set to false, blank popups are opened before anything else happens. When set to true, popups are opened when making the network request. Can be set to true for scenarios where `about:blank` is not supported, e.g. desktop apps or progressive web apps | boolean                              | `false`                                          |
+| `navigatePopups`           | Sets whether popups are opened and navigated immediately. By default, this flag is set to false. When set to false, blank popups are opened before anything else happens. When set to true, popups are opened when making the network request and navigation occurs immediately. Can be set to true for scenarios where `about:blank` is not supported, e.g. desktop apps or progressive web apps | boolean | `false` |
```

Comment:
The options here are reverse of what I would expect based on the name - navigatePopups = true -> about:blank **navigates to** login domain. navigatePopups = false -> Popup opened directly to login domain, no navigation